crates/ruff/src/rules/pyupgrade/snapshots/ruff__rules__pyupgrade__tests__UP006.py.snap

```diff
@@ -1,13 +1,13 @@
 ---
 source: crates/ruff/src/rules/pyupgrade/mod.rs
 ---
-UP006.py:4:10: UP006 [*] Use `list` instead of `List` for type annotations
+UP006.py:4:10: UP006 [*] Use `list` instead of `typing.List` for type annotation
   |
 4 | def f(x: typing.List[str]) -> None:
```

Comment:
Should this fix remove the `typing` import if it was the last usages or do we leave this to the unused imports lint rule?

Reply:
We leave it to the unused imports lint.